espanol/canudos.md

```diff
@@ -199,3 +199,5 @@ Luego de su viaje extraordinario se acordó de su [máquina del tiempo](maquina/
 
 Y despues de varios disparos decidio ir a comer, mientras su amigo lo cubria el encendia el coche
 para iniciar su viaje directo al [espacio](space/space.md).
+
+Lo que le sirvieron en el restaurante fueros unos [macarrones con tomatito del lidl, queso y choricito](macarrones/macarras.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.